core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClientBuilder.java

```diff
@@ -28,8 +30,12 @@
 public class RetryingHttpClientBuilder extends RetryingClientBuilder<
         RetryingHttpClientBuilder, RetryingHttpClient, HttpRequest, HttpResponse> {
 
+    private static final int DEFAULT_CONTENT_PREVIEW_LENGTH = 1024 * 1024; // 1 MiB
```

Comment:
I was just afraid of having too much variables in `Flags`, and I though that I don't have to move this into `Flags`. How about moving it later when I'm going to be sure that it need to be moved?